ec2-dash/abnormal_hr.py

```diff
@@ -31,5 +34,117 @@ def compare_hr_to_max_hr(hr: str, max_hr: int) -> bool:
     
     return False
 
-# initial plan was to send to lambda then the lambda can talk to the SNS but seems 
-# that the EC2 can do this so will talk with the team in the morning 
\ No newline at end of file
+def create_dict_for_email(user_info: dict, curr_hr: int, max_hr:int, date: str, time_elapsed: str) -> dict:
```

Comment:
Potentially change this function name if you are also invoking the send email message within it as well. 

Return the dict to a variable in dash_sqlite.py and then invoke the send_user_hr_warning on its own,
Or import the whole module into  dash_sqlite.py and run both functions with `if name == __main__:` in this module